content/docs/1.6.0/best-practices.md

```diff
@@ -117,14 +120,20 @@ Longhorn system will create snapshots automatically when rebuilding a replica. R
 
 We recommend setting the CPU request for Longhorn instance manager pods.
 
-The `Guaranteed Instance Manager CPU` setting allows you to reserve a percentage of a node's total allocatable CPU for all instance manager pods.
+### V1 Data Engine
+
+The `Guaranteed Instance Manager CPU` setting allows user to reserver percentage of the total allocatable CPU resources on each node for each instance manager pod when the V1 Data Engine is enabled. The default value is 12 percent.
```

Comment:
```suggestion
The `Guaranteed Instance Manager CPU` setting allows you to reserve a percentage of the total allocatable CPU resources on each node for each instance manager pod when the V1 Data Engine is enabled. The default value is 12.
```

Reply:
When we mention default values, we are referring to the actual user input. In this case, the user is only required to type a number (without the % symbol).